boot/aether/src/boot/aether.clj

```diff
@@ -68,7 +68,7 @@
 
 (defn transfer-listener
   [info]
-  (util/dbug "Aether: %s\n" (with-out-str (pprint/pprint info)))
+  (util/trace* "Aether: %s\n" (with-out-str (pprint/pprint info)))
```

Comment:
FYI: 
This should fix the bad network throughput. (ref. #565)
In my environment, it takes about 10 minutes to fetch 20MB jar file with `dbug`, while it takes only 20 seconds with `dbug*`. I tried to make a PR by myself but I will wait for this PR to be merged.
BTW, it seems that this code was introduced by [this conflict resolution commit](https://github.com/boot-clj/boot/commit/de16fb099dac2795a5e314038e5edf2d07b92807#diff-b547159e61e59485c59eccc8b7d5f7a1).